builtins/amp-img.js

```diff
@@ -197,6 +213,13 @@ export class AmpImg extends BaseElement {
 
     this.img_.setAttribute('src', src);
 
+    if (this.useNativeSrcset_) {
```

Comment:
Don't do all the work above if going into this branch. I expected that you needed to generated a `sizes` attribute if the user provided none.

Reply:
+1 even better if we only parse and create `srcset_` if 1- `src` is not provided and 2- browser does not support `srcset` (even if the check is a simple if IE <= 11)